EleksTubeHAX_pio/src/main.cpp

```diff
@@ -49,21 +48,24 @@ uint8_t       yesterday       = 0;
 // Helper function, defined below.
 void updateClockDisplay(TFTs::show_t show=TFTs::yes);
 void setupMenu(void);
-void EveryFullHour(bool loopUpdate=false);
```

Comment:
I know it's hard to resist, but please let's just make functional changes in the first step and not make the source code prettier yet.